cli/global_state.rs

```diff
@@ -134,37 +132,17 @@ impl GlobalState {
       builder.insert(&module_specifier).await?;
       let mut graph = builder.get_graph(&self.lockfile)?;
 
-      // TODO(kitsonk) this needs to move, but CompilerConfig is way too
-      // complicated to use here.
-      let maybe_config = if let Some(path) = self.flags.config_path.clone() {
-        let cwd = std::env::current_dir()?;
-        let config_file = cwd.join(path);
-        let config_path = config_file.canonicalize().map_err(|_| {
-          io::Error::new(
-            io::ErrorKind::InvalidInput,
-            format!(
-              "Could not find the config file: {}",
-              config_file.to_string_lossy()
-            ),
-          )
-        })?;
-        let config_str = fs::read_to_string(config_path)?;
-
-        Some(config_str)
-      } else {
-        None
-      };
-
       let (stats, maybe_ignored_options) =
         graph.transpile(TranspileOptions {
           debug: self.flags.log_level == Some(log::Level::Debug),
-          maybe_config,
+          maybe_config_path: self.flags.config_path.clone(),
```

Comment:
The only problem is see with this change is that `runtime_transpile` will require specialized version (because config is passed as JSON object). I guess it could be addressed later.

Reply:
Yeah, that is a valid point, but yes, let's address later, as I think it will be easier to figure out how best to abstract that out.